### PR TITLE
Get a list of RICE sorted issues and PRs

### PR DESCRIPTION
**Pull request by thinkingserious**
This PR allows you to retrieve a RICE sorted list of all open issues and PRs in the specified repos.

### CODE REVIEW

File: services/tasks/project/api/routes.py [hunk 1]
```diff
@@ -125,42 +115,30 @@ def get_all_tasks():
...
-                    updated_at = issue['updatedAt']
-                    labels = issue['labels']
-                    num_of_comments = issue['comments']
-                    num_of_reactions = issue['reactions']
-                    title = issue['title']
-                    language = repo['programming_language']
+        if len(repos[repo['name']]) != 0:
+            for item in repos[repo['name']]:
+                if item != None:
+                    task_type = 'pr' if 'pull' in item['url'] else 'issue'
```
Comment (childish-sambino) [inline services/tasks/project/api/routes.py]: Hopefully no repos have `pull` in their name (:

File: services/tasks/project/api/routes.py [hunk 2]
```diff
@@ -125,42 +115,30 @@ def get_all_tasks():
...
-                                num_of_reactions=num_of_reactions,
-                                title=title,
-                                url=url,
+                                created_at=item['createdAt'],
+                                updated_at=item['updatedAt'],
+                                creator=item['author'],
+                                labels=item['labels'],
+                                language=repo['programming_language'],
+                                num_of_comments=item['comments'],
+                                num_of_reactions=item['reactions'],
```
Comment (childish-sambino) [inline services/tasks/project/api/routes.py]: Should these be `item['num_of_comments']`?

File: services/tasks/project/api/routes.py (hunk 1 above)
Comment (thinkingserious) [reply services/tasks/project/api/routes.py]: Hmm, I think if I rename `repos` to `items` that should lessen the confusion.

File: services/tasks/project/api/routes.py (hunk 2 above)
Comment (thinkingserious) [reply services/tasks/project/api/routes.py]: It's the right value, but I should change the naming. Thanks!

File: services/tasks/project/api/routes.py (hunk 1 above)
Comment (childish-sambino) [reply services/tasks/project/api/routes.py]: But the URL for an issue could have `pull` in it: `https://github.com/sendgrid/dx-pulley/issues/62`

Comment (thinkingserious) [reply services/tasks/project/api/routes.py]: Ah, gotcha! I should look for `/pull/` instead :)

**Comment by thinkingserious** [comment]
Hello @thinkingserious,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**